ophyd/signal.py

```diff
@@ -70,6 +71,14 @@ def __init__(self, *, name, value=0., timestamp=None, parent=None,
         # self.tolerance is a property
         self.rtolerance = rtolerance
 
+        # Signal defaults to being connected, with full read/write access.
```

Comment:
Hm. I could argue this in both directions. What's your reasoning?

Reply:
As-is, `Signal` can be used as a soft signal. Because of that, you generally want the ability to read, write, and say that it's "connected" straightaway.

We are cheating in a way here by not having a `SignalBase` that defaults to all of these being `False` and raising `NotImplementedError` for all default implementations. I'm 50/50 on whether this is actually a deficiency or a fair removal of one probably-unnecessary level of the class hierarchy...

Reply:
Makes sense. I forget who said, "In Python, inheritance is just for code reuse," but in that spirit I'm happy with the current situation. We don't have much code to reuse.